index.html

```diff
@@ -0,0 +1,4 @@
+<link href="index.css" rel="stylesheet">
+<header>
+  <h1>Hello Microverse!</h1>
+</header>
```

Comment:
- [ ] Kindly use a proper HTML boilerplate, and make sure to style the header with class. 😄